web-wallet/src/routes/(app)/dashboard/transactions/__tests__/page.spec.js

```diff
@@ -1,15 +1,25 @@
-import { afterEach, describe, expect, it } from "vitest";
+import { afterEach, describe, expect, it, vi } from "vitest";
 import { cleanup, render } from "@testing-library/svelte";
 import Transactions from "../+page.svelte";
 
+global.ResizeObserver = vi.fn().mockImplementation(() => ({
```

Comment:
Why we need this?
The `ResizeObserver` is already "mocked" in `vite.setup.js`.

Reply:
<img width="1146" alt="Screenshot 2024-02-05 at 17 09 27" src="https://github.com/dusk-network/rusk/assets/34094428/5ced0713-c5a0-4c24-abf5-41580a23839d">

Not sure, the test fails for me without it. Is something extra needed to make the test recognize the mock in the vite-setup? I wouldn't expect it to be the case as it should be on the global object...

Reply:
Ok, then some extra investigation is needed.

Leave it like it was for now, as I see there is already another place where a manual mock is created: we should open an issue to tackle this.

Reply:
Have opened an issue #1351.